test/verify/check-pages

```diff
@@ -809,6 +809,16 @@ OnCalendar=daily
             b.switch_to_top()
             assert_location("/system")
 
+    def testAllNavEntries(self):
+        b = self.browser
+        self.login_and_go()
+
+        # the <a> links should be unique by their href= attributes, so get these
+        hrefs = b.eval_js("[...document.querySelectorAll('#nav-system .nav-item a')].map(el => el.getAttribute('href'))")
```

Comment:
I always wonder why the DOM API keeps living in the past :)

Reply:
Like, giving you this crappy `NodeList` which isn't an actual array? yeah..